docs/RFCS/explain_debug.md

```diff
@@ -0,0 +1,71 @@
+- Feature Name: explain
+- Status: draft
+- Start Date: 2015-08-10
+- RFC PR:
+- Cockroach Issue: #2047
+
+# Summary
+
+Add support for an `EXPLAIN DEBUG` statement. The `EXPLAIN DEBUG` statement
```

Comment:
The statement is `EXPLAIN` and `DEBUG` is an option similar to `EXPLAIN VERBOSE`. Not sure about your other statement as I'm certainly using `EXPLAIN DEBUG` in the example at line 41.